workitem/workitemtype_repository.go

```diff
@@ -111,8 +112,9 @@ func ClearGlobalWorkItemTypeCache() {
 
 // Create creates a new work item in the repository
 // returns BadParameterError, ConversionError or InternalError
-func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition) (*WorkItemType, error) {
+func (r *GormWorkItemTypeRepository) Create(ctx context.Context, spaceID uuid.UUID, id *uuid.UUID, extendedTypeID *uuid.UUID, name string, description *string, icon string, fields map[string]FieldDefinition, categories []*uuid.UUID) (*WorkItemType, error) {
```

Comment:
> Do we have to add the category during WIT creation?]

No, you can just pass empty array and it will not associate WIT with any category (https://github.com/almighty/almighty-core/pull/1037/files/#diff-33431f309121c7a29ce1a930347affabR93)

> Can't we just add it later?

Okay, So basically there is no "Update workitemtype" action. Also, WIT are associated with category during space template creation. 

> Also, technically speaking, a WIT doesn't need a category to exist.
True, I have handled this scenario in my PR.